Failure during 'make test'

```
  CPAN.pm: Building G/GE/GENEHACK/Git-Wrapper-0.030.tar.gz

Checking if your kit is complete...
Looks good
Writing Makefile for Git::Wrapper
Writing MYMETA.yml and MYMETA.json
cp lib/Git/Wrapper/File/RawModification.pm blib/lib/Git/Wrapper/File/RawModification.pm
cp lib/Git/Wrapper.pm blib/lib/Git/Wrapper.pm
cp lib/Git/Wrapper/Statuses.pm blib/lib/Git/Wrapper/Statuses.pm
cp lib/Git/Wrapper/Log.pm blib/lib/Git/Wrapper/Log.pm
cp lib/Git/Wrapper/Exception.pm blib/lib/Git/Wrapper/Exception.pm
cp lib/Git/Wrapper/Status.pm blib/lib/Git/Wrapper/Status.pm
Manifying blib/man3/Git::Wrapper::Exception.3
Manifying blib/man3/Git::Wrapper::Log.3
Manifying blib/man3/Git::Wrapper.3
Manifying blib/man3/Git::Wrapper::File::RawModification.3
Manifying blib/man3/Git::Wrapper::Statuses.3
Manifying blib/man3/Git::Wrapper::Status.3
  GENEHACK/Git-Wrapper-0.030.tar.gz
  /usr/bin/make -- OK
Running make test
PERL_DL_NONLAZY=1 /usr/local/bin/perl "-MExtUtils::Command::MM" "-e" "test_harness(0, 'blib/lib', 'blib/arch')" t/*.t
t/00-load.t ................ 6/6 # Testing Git::Wrapper 0.030
t/00-load.t ................ ok   
t/000-requires_external.t .. ok   
t/author-err.t ............. skipped: these tests are for testing by the author
t/basic.t .................. # Testing git version: 1.6.3.2
t/basic.t .................. ok    
t/git_binary.t ............. ok   
t/parse_args.t ............. ok    
t/path_class.t ............. # Testing git version: 1.6.3.2
t/path_class.t ............. 2/? fatal: '/tmp/FRA4lgN3mK/baz' is outside repository
# Tests were run but no plan was declared and done_testing() was not seen.
t/path_class.t ............. Dubious, test returned 128 (wstat 32768, 0x8000)
All 2 subtests passed 
t/release-pod-coverage.t ... skipped: these tests are for release candidate testing
t/release-pod-syntax.t ..... skipped: these tests are for release candidate testing

Test Summary Report
-------------------
t/path_class.t           (Wstat: 32768 Tests: 2 Failed: 0)
  Non-zero exit status: 128
  Parse errors: No plan found in TAP output
Files=9, Tests=62, 21 wallclock secs ( 0.24 usr  0.14 sys +  3.21 cusr  2.91 csys =  6.50 CPU)
Result: FAIL
Failed 1/9 test programs. 0/62 subtests failed.
make: *** [test_dynamic] Error 255
  GENEHACK/Git-Wrapper-0.030.tar.gz
  /usr/bin/make test -- NOT OK
//hint// to see the cpan-testers results for installing this module, try:
  reports GENEHACK/Git-Wrapper-0.030.tar.gz
Running make install
  make test had returned bad status, won't install without force
Failed during this command:
 GENEHACK/Git-Wrapper-0.030.tar.gz            : make_test NO
install
```
```
$ perl -V
Summary of my perl5 (revision 5 version 18 subversion 0) configuration:
   
  Platform:
    osname=darwin, osvers=8.11.0, archname=darwin-2level
    uname='darwin macintosh-9.local 8.11.0 darwin kernel version 8.11.0: wed oct 10 18:26:00 pdt 2007; root:xnu-792.24.17~1release_ppc power macintosh powerpc '
    config_args='-des'
    hint=recommended, useposix=true, d_sigaction=define
    useithreads=undef, usemultiplicity=undef
    useperlio=define, d_sfio=undef, uselargefiles=define, usesocks=undef
    use64bitint=undef, use64bitall=undef, uselongdouble=undef
    usemymalloc=n, bincompat5005=undef
  Compiler:
    cc='cc', ccflags ='-fno-common -DPERL_DARWIN -fno-strict-aliasing -pipe -I/usr/local/include -I/opt/local/include',
    optimize='-O3',
    cppflags='-fno-common -DPERL_DARWIN -fno-strict-aliasing -pipe -I/usr/local/include -I/opt/local/include'
    ccversion='', gccversion='4.0.1 (Apple Computer, Inc. build 5250)', gccosandvers=''
    intsize=4, longsize=4, ptrsize=4, doublesize=8, byteorder=4321
    d_longlong=define, longlongsize=8, d_longdbl=define, longdblsize=16
    ivtype='long', ivsize=4, nvtype='double', nvsize=8, Off_t='off_t', lseeksize=8
    alignbytes=8, prototype=define
  Linker and Libraries:
    ld='env MACOSX_DEPLOYMENT_TARGET=10.3 cc', ldflags =' -L/usr/local/lib -L/opt/local/lib'
    libpth=/usr/local/lib /opt/local/lib /usr/lib
    libs=-ldbm -ldl -lm -lc
    perllibs=-ldl -lm -lc
    libc=, so=dylib, useshrplib=false, libperl=libperl.a
    gnulibc_version=''
  Dynamic Linking:
    dlsrc=dl_dlopen.xs, dlext=bundle, d_dlsymun=undef, ccdlflags=' '
    cccdlflags=' ', lddlflags=' -bundle -undefined dynamic_lookup -L/usr/local/lib -L/opt/local/lib'


Characteristics of this binary (from libperl): 
  Compile-time options: HAS_TIMES PERLIO_LAYERS PERL_DONT_CREATE_GVSV
                        PERL_HASH_FUNC_ONE_AT_A_TIME_HARD PERL_MALLOC_WRAP
                        PERL_PRESERVE_IVUV PERL_SAWAMPERSAND USE_LARGE_FILES
                        USE_LOCALE USE_LOCALE_COLLATE USE_LOCALE_CTYPE
                        USE_LOCALE_NUMERIC USE_PERLIO USE_PERL_ATOF
  Built under darwin
  Compiled at May 18 2013 12:31:26
  %ENV:
    PERLBREW_BASHRC_VERSION="0.59"
    PERLBREW_HOME="/Users/jimk/.perlbrew"
    PERLBREW_MANPATH=""
    PERLBREW_PATH="/Users/jimk/perl5/perlbrew/bin"
    PERLBREW_ROOT="/Users/jimk/perl5/perlbrew"
    PERLBREW_VERSION="0.59"
  @INC:
    /usr/local/lib/perl5/site_perl/5.18.0/darwin-2level
    /usr/local/lib/perl5/site_perl/5.18.0
    /usr/local/lib/perl5/5.18.0/darwin-2level
    /usr/local/lib/perl5/5.18.0
    /usr/local/lib/perl5/site_perl/5.16.0
    /usr/local/lib/perl5/site_perl/5.14.2
    /usr/local/lib/perl5/site_perl/5.14.0
    /usr/local/lib/perl5/site_perl/5.12.0
    /usr/local/lib/perl5/site_perl/5.10.1
    /usr/local/lib/perl5/site_perl/5.10.0
    /usr/local/lib/perl5/site_perl/5.8.6
    /usr/local/lib/perl5/site_perl
    .
```

--- a/t/path_class.t
+++ b/t/path_class.t
@@ -8,11 +8,18 @@ use POSIX qw(strftime);
 use Sort::Versions;
 use Test::Deep;
 use Test::Exception;
+use File::Spec;
+use Cwd qw/abs_path/;
+
+my $DO_WIN32_GETLONGPATHNAME = ($^O eq 'MSWin32') ? eval 'use Win32; 1' : 0;
 
 eval "use Path::Class 0.26; 1" or plan skip_all =>
     "Path::Class 0.26 is required for this test.";
 
-my $tempdir = tempdir(CLEANUP => 1);
+my $tmpdir = File::Spec->tmpdir;
+$tmpdir = Win32::GetLongPathName(abs_path($tmpdir)) if $DO_WIN32_GETLONGPATHNAME;
+
+my $tempdir = tempdir(DIR => $tmpdir, CLEANUP => 1);
 
 my $dir = Path::Class::dir($tempdir);
 

